### PR TITLE
Added linux/amd64 C2D, also fixed up Makefile to use patch.

### PR DESCRIPTION
**Pull request by AeroNotix**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -35,7 +35,7 @@ $(GO_CHECKOUT):
 
 $(GO_103_ROOT): $(GO_CHECKOUT)
 	hg clone -b release-branch.go1 $(GO_CHECKOUT) $@
-	hg import --cwd $@ --no-commit $(TOP)/patches/6501099.diff
+	hg import --cwd $@ --no-commit $(TOP)/patches/6501099.diff $(TOP)/patches/7665047.diff
```
Comment (davecheney) [inline Makefile]: please keep this on two lines, if there are other patches, this line will get unwieldy

**Comment by davecheney** [comment]
Please revert the change to the Makefile then I will merge this

**Comment by AeroNotix** [comment]
Reverted

**Comment by davecheney** [comment]
Marvelous, thank you.

**Comment by AeroNotix** [comment]
A pleasure.

**Pull request merged**